test/LTE_Tests/runLTETests.m

```diff
@@ -54,10 +54,19 @@
         otherwise
             error('%s unsupported for LTE test harness', BoardName);
     end
-    params = Parameter.fromData('AD936xClassTx',AD936xClassTx, ...
-        'AD936xClassRx',AD936xClassRx, ...
+    % check that hardware is connected
+    params = Parameter.fromData('AD936xClassTx', AD936xClassTx, ...
+        'AD936xClassRx', AD936xClassRx);
+    suite = TestSuite.fromMethod(?AD936x_LTETests, 'CheckForHardware', ...
+        'ExternalParameters', params);
+    run(suite);
```

Comment:
If the test fails what happens? Does it just keep running?

Reply:
Addressed by the latest commit. Test is labeled incomplete.